test/UnitTests/Controllers/RestaurantControllerUnitTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using Moq;
+using AskToniApi.Controllers;
+using AskToniApi.Models;
+
+namespace AskToniApi.UnitTests.Controllers
+{
+    public class RestaurantControllerUnitTests
+    {
+        [Fact]
+        public void Get_Success_ReturnsCorrectResult()
+        {
+            var mockRecommendationRepository = new Mock<IRecommendationRepository>();
+            var restaurantCollection = new List<Restaurant>();
+            restaurantCollection.Add(new Restaurant() 
+                                        { 

```

Comment:
You might want to consider enforcing some style rules on the code, this can be easily automated with tools too!
I did a quick browse and this looks interesting: [https://github.com/DotNetAnalyzers/StyleCopAnalyzers](https://github.com/DotNetAnalyzers/StyleCopAnalyzers)

I already have a plugin called [https://github.com/eslint/eslint](https://github.com/eslint/eslint) on the client, and for the bot @edison-cy-yang setup which does the same, in our case it automatically formats the code when we do a git push.

Reply:
Not a bad idea, thanks. Maybe we can add it as an issue?